prov/tcp/src/tcpx_ep.c

```diff
@@ -217,6 +217,7 @@ tcpx_ep_accept(struct fid_ep *ep_fid, const void *param, size_t paramlen)
 	    !handle || (handle->fid.fclass != FI_CLASS_CONNREQ))
 		return -FI_EINVAL;
 
+	free(handle);
```

Comment:
The thinking here is if fi_accept() fails, the application will call fi_reject().  So the handle is only freed on success.